README.md

```diff
@@ -61,6 +61,7 @@ addon | summary
 [spp_entitlement_basket](spp_entitlement_basket/) | This module allows you to define baskets of goods and services that beneficiaries are entitled to receive, simplifying in-kind entitlement management within social protection programs.
 [spp_entitlement_cash](spp_entitlement_cash/) | Manage cash-based entitlements for beneficiaries within social protection programs, including defining calculation rules, automating disbursement, and tracking payments.
 [spp_entitlement_in_kind](spp_entitlement_in_kind/) | Manages the distribution of in-kind entitlements within social protection programs, handling inventory, service points, and beneficiary redemption.
+[spp_ethnic_group](spp_ethnic_group/) | Manages ethnic group
```

Comment:
@jeremi @dasunhegoda please elaborate on this description if needed, or let me know if we can release with the existing description I put there during the preparation of this PR

Reply:
I asked @reichie020212, @Markpenalosa, and @emjay0921. Couldn't find a better desc. Maybe @gonzalesedwin1123 knows. If not, let's proceed with this.

Reply:
Let's proceed "as-is".